<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sanifica_Window</class>
 <widget class="QWidget" name="Sanifica_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sanifica</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QComboBox" name="prodotti_comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox::down-arrow {


        image: url(:/icons/res/sanifica_icon.png);
        width: 14px;
        height: 14px;

        }
        QComboBox::drop-down
        {
        border: 0px; /* This seems to replace the whole arrow of the combo box */
        color:black

        }

QComboBox {
    color: black;
border-radius:15px;
border: 5px solid rgb(0,140,255);
	selection-color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtTop</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="metri_cubi_spinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox::down-button {

        image: url(:/arrows/res/down_arrow.png);
        width:50px;


        }


        QSpinBox::up-button {


        image: url(:/arrows/res/up_arrow.png);
        width: 50px;
        }


        QSpinBox {
        color: rgb(0, 0, 0);
        border-radius:25px;
	background-color: rgb(255, 255, 255);
border: 5px solid rgb(0,140,255);


        }</string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="sanifica_btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);

color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Sanifica</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-10</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Seleziona Prodotto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/res/select.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Metri Cubi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);
                    color: rgb(255, 255, 255);
                    border-radius:10px;
                </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img_resources.qrc">
     <normaloff>:/cancel/res/cancel.png</normaloff>:/cancel/res/cancel.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);
                 color: rgb(255, 255, 255);
                 border-radius:10px;
             </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img_resources.qrc">
     <normaloff>:/cancel/res/cancel.png</normaloff>:/cancel/res/cancel.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancel_btn_</sender>
   <signal>clicked()</signal>
   <receiver>Sanifica_Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
